railties/lib/rails/app_updater.rb

```diff
@@ -31,7 +31,6 @@ def generator_options
           options[:skip_test]           = !defined?(Rails::TestUnitRailtie)
           options[:skip_asset_pipeline] = !defined?(Sprockets::Railtie) && !defined?(Propshaft::Railtie)
           options[:skip_bootsnap]       = !defined?(Bootsnap)
-          options[:updating]            = true
```

Comment:
I initially picked `:update` because its older, but I also think it aligns with the naming of other options (since these are technically `rails new` CLI options). `:pretend`, `:force`, `:quiet` etc. make me think `:update` is a better fit vs `:pretending`, `:forcing`, `:quietly` where I think `:updating` would make more sense.

Reply:
Ok, that makes sense! :+1: